tools/StaticAnalysis/Exceptions/BreakingChangeIssues.csv

```diff
@@ -1144,6 +1144,9 @@
 "Microsoft.Azure.Commands.Compute.dll","Microsoft.Azure.Commands.Compute.Automation.SetAzureRmVmssStorageProfileCommand","Set-AzureRmVmssStorageProfile","0","3030","The generic type for 'parameter ManagedDisk' has been changed from 'System.Nullable`1[Microsoft.Azure.Management.Compute.Models.StorageAccountTypes]' to 'System.Nullable`1[Microsoft.Azure.Commands.Compute.Models.PSStorageAccountTypes]'. ","Change the generic type for 'parameter ManagedDisk' back to 'System.Nullable`1[Microsoft.Azure.Management.Compute.Models.StorageAccountTypes]'."
 "Microsoft.Azure.Commands.Subscription.dll","Microsoft.Azure.Commands.Subscription.Cmdlets.NewAzureRmSubscriptionDefinition","New-AzureRmSubscriptionDefinition","0","1000","The cmdlet 'New-AzureRmSubscriptionDefinition' has been removed and no alias was found for the original cmdlet name.","Add the cmdlet 'New-AzureRmSubscriptionDefinition' back to the module, or add an alias to the original cmdlet name."
 "Microsoft.Azure.Commands.Subscription.dll","Microsoft.Azure.Commands.Subscription.Cmdlets.GetAzureRmSubscriptionDefinition","Get-AzureRmSubscriptionDefinition","0","1000","The cmdlet 'Get-AzureRmSubscriptionDefinition' has been removed and no alias was found for the original cmdlet name.","Add the cmdlet 'Get-AzureRmSubscriptionDefinition' back to the module, or add an alias to the original cmdlet name."
+"Microsoft.Azure.Commands.Resources.dll","Microsoft.Azure.Commands.Resources.GetAzureRoleDefinitionCommand","Get-AzureRmRoleDefinition","0","2000","The cmdlet 'Get-AzureRmRoleDefinition' no longer supports the parameter 'AtScopeAndBelow' and no alias was found for the original parameter name.","Add the parameter 'AtScopeAndBelow' back to the cmdlet 'Get-AzureRmRoleDefinition', or add an alias to the original parameter name."
```

Comment:
These three entries probably should be removed, as they aren't included in this PR.